lib/MusicBrainz/Server/Controller/WS/2/Release.pm

```diff
@@ -190,6 +188,18 @@ sub release_toplevel {
         }
     }
 
+    if (@ac_entities) {
+        $c->model('ArtistCredit')->load(@ac_entities);
+
+        my @acns = map { $_->all_names } uniq_by { refaddr($_) }
```

Comment:
What's the benefit of uniq_by refaddr rather than just uniq here?

Reply:
Don't think there was a reason there. Changed it.